src/background/responders.ts

```diff
@@ -101,7 +101,7 @@ export const responders: Responders<Action> = {
     return updateTabFeedbackIfExists(state, tabId, tab => {
       if (tab.domain !== domain) return {}
       return {
-        editorState: replaceHandle(tab.feedbackState.editorState, handle),
+        editorState: handle ? replaceHandle(tab.feedbackState.editorState, handle) : tab.feedbackState.editorState,
```

Comment:
If the handle is `null` we should delete the old handle if it exists. I think we should do:
`editorState: replaceHandle(tab.feedbackState.editorState, handle || ''),`
Note the `|| ''`

Reply:
Oh yes, you are right